src/Pos/Launcher/Scenario.hs

```diff
@@ -97,11 +95,10 @@ waitForPeers = discoverPeers >>= \case
 initSemaphore :: (WorkMode ssc m) => m ()
 initSemaphore = do
     semaphore <- ncBlkSemaphore <$> getNodeContext
-    unlessM
-        (liftIO $ isEmptyMVar semaphore)
-        (logError "ncBlkSemaphore is not empty at the very beginning")
+    whenNothingM_ (tryReadMVar semaphore) $
```

Comment:
Why is it better than `unlessM (isEmptyMVar semaphore)`?

Ok, I see that `isEmptyMVar` is not in `Universum`. Why is it worse than other `*MVar` functions?